maqs-webservices/src/test/java/com/cognizantsoftvision/maqs/webservices/WebServiceDriverRequestUnitTest/WebServiceDriverPutUnitTest.java

```diff
@@ -22,7 +18,7 @@
 /**
  * The Web Driver Service Driver Put unit test class.
  */
-public class WebServiceDriverPutUnitTest {
+public class WebServiceDriverPutUnitTest extends BaseWebServiceTest {
```

Comment:
Why are we extending base service if it is not required for the tests?

Reply:
changed back to baseGenericTest